README.md

```diff
@@ -1,155 +1,134 @@
+- [Build Instructions](#build-instructions)
+- [Overview](#overview)
+- [FAQ](#faq)
+- [Support](#support)
+
+## License
+MobileCoin is available under open-source licenses. Look for the *LICENSE* file in each crate for more information.
+
+## Cryptography Notice
+This distribution includes cryptographic software. Your country may have restrictions on the use of encryption software. Please check your country's laws before downloading or using this software.
+
+## Repository Structure
+|Directory |Description |
+| :-- | :-- |
+| [attest](./attest) | Remote attestation primitives. |
+| [build-info](./build-info) | Measurements made at compile time. |
+| [common](./common) | Items shared across MobileCoin crates. |
+| [consensus](./consensus) | Byzantine Fault Tolerant Consensus. |
+| [crypto](./crypto) | Cryptography. |
+| [enclave-boundary](./enclave-boundary) | Intel® SGX ECALL infrastructure. |
+| [ledger](./ledger) | Storage and synchronization for the MobileCoin blockchain. |
+| [mcbuild](./mcbuild/) | Tools for building and signing enclaves. |
+| [mcconnection](./mcconnection/) | Attested MobileCoin connections. |
+| [mobilecoind](./mobilecoind/) | Blockchain daemon and example client code. |
+| [peers](./peers/) | Peer-to-peer networking. |
+| [sgx](./sgx/) | Support for Intel® Software Guard eXtensions (Intel SGX). |
+| [transaction](./transaction/) | Private transactions. |
+| [util](./util/) | Testing and bootstrap utilities. |
+
+#### Selected Binaries
+| Target | Description | Nodes |
+| :-- | :-- |:--|
+| [`consensus-service`](./consensus/service) | Validates new transactions for the public ledger.| Validator Nodes |
+| [`ledger-distribution`](./ledger/distribution) | Publishes the ledger to long-term storage. | Full Validator Nodes|
+| [`mobilecoind`](./mobilecoind) | Synchronizes the ledger and provides the desktop API. | Watcher and Validator Nodes |
+
+## Build Instructions
+
+The workspace can be built with `cargo build` and tested with `cargo test`. Either command will recognize the cargo `--release` flag to build with optimizations.
+
+Some crates (for example [`consensus-service`](./consensus/service)) depend on Intel SGX, which adds additional build and runtime requirements. For detailed information about setting up a build environment, how enclaves are built, and on configuring the build, see [BUILD.md](BUILD.md).
+
+For a quick start, you can build in the same docker image that we use for CI, using the `mob` tool. Note that this requires you to install [Docker](https://docs.docker.com/get-docker/). You can use the `mob` tool with the following commands:
 
 ```
-# From the root of the repo
+# From the root of the cloned repository
 ./mob prompt
 
-# At the container prompt
+# At the resulting docker container prompt
 cargo build
 ```
 
-#### License
-
-The components of MobileCoin require different licenses. Look for the LICENSE file in each crate for more information.
-
-#### Overview
+## Overview
```

Comment:
What do you think about moving this out into `OVERVIEW.md` and linking to it from main `README`? Then our README will be more slim

Reply:
I'm not opposed if a slim README is a goal. I'm also happy with it as it is now.